route_following_plugin/src/route_following_plugin.cpp

```diff
@@ -46,7 +46,7 @@ namespace route_following_plugin
         
         pnh_->param<double>("minimal_maneuver_duration", mvr_duration_, 16.0);
         pnh2_->param<double>("config_speed_limit",config_limit);
-        pnh_->param<double>("route_end_jerk", jerk_);
+        pnh_->param<double>("/guidance/route_end_jerk", jerk_, 1.0);
```

Comment:
This parameter loading pattern still seems a bit weird to me. Is there a reason why only this param is getting mapped into the guidance namespace? Or why we have 2 different private node handles (`pnh_` and `pnh2_`) doing this loading? 

Looking at it further, I'd wager that all of route_following_plugins params are being misloaded, they're being put into this guidance namespace instead of under the route_following_plugin's own namespace like they should be. Compare how route_following's launch file looks:
```
    <rosparam command="load" file="$(find route_following_plugin)/config/parameters.yaml"/>
    <node name="route_following_plugin" pkg="route_following_plugin" type="route_following_plugin" output="log"/>
```
to how trajectory_executor is:
```
<node pkg="trajectory_executor" type="trajectory_executor_node" name="trajectory_executor_node">
        <rosparam command="load" file="$(find trajectory_executor)/config/trajectory_executor.yaml" />
    </node>
```

With the param loading outside vs inside the node tag, I think that's probably putting all of route_following's params in the /guidance/ namespace. Let's plan to fix this on develop, but if there are any other of these params that might be giving us issues in the release let's see what we can do to fix it here with minimal breakage risk (by changing the param like you have here)

Reply:
Yes, this is what i think is happening too. With the other params, its not giving an issue because the default values match the values in the config file.

Reply:
Okay, if the other params are fine for now we can leave it and plan to fix it on develop. Let's make an issue to track this.

Reply:
https://github.com/usdot-fhwa-stol/carma-platform/issues/1157